alternator/serialization.cc

```diff
@@ -319,6 +323,19 @@ std::optional<big_decimal> try_unwrap_number(const rjson::value& v) {
     }
 }
 
+std::optional<bytes> unwrap_bytes(const rjson::value& value, bool from_query) {
+    bytes b;
+    try {
+        b = rjson::base64_decode(value);
```

Comment:
I think you can greatly simplify this function by doing here
```
return rjson::base64_decode(value);
```
then you don't need the `bytes b` or the `return std::move(b)`.

Reply:
yes, that's true